exampleCourse/questions/element/multipleChoice/question.html

```diff
@@ -1,263 +1,297 @@
-               In particular, all options are presented in a randomized form. Only one option may be selected from the list. </p>
-            <p> What is the color of the sky? </p>
-        </pl-question-panel>
-
-        <pl-multiple-choice answers-name="sky-color1">
-          <pl-answer correct="true">Blue</pl-answer>
-          <pl-answer>Pink</pl-answer>
-          <pl-answer>Purple</pl-answer>
-          <pl-answer>Orange</pl-answer>
-          <pl-answer>Yellow</pl-answer>
-          <pl-answer>Brown</pl-answer>
-          <pl-answer>Red</pl-answer>
-        </pl-multiple-choice>
-
-    </div>
+  <div class="card-header">Part 1</div>
+
+  <div class="card-body">
+    <pl-question-panel>
+      <p>
+        Here, you can observe the default behavior of the <code>pl-multiple-choice</code> element.
+        In particular, all options are presented in a randomized form. Only one option may be
+        selected from the list.
+      </p>
+      <p>What is the color of the sky?</p>
+    </pl-question-panel>
+
+    <pl-multiple-choice answers-name="sky-color1">
+      <pl-answer correct="true">Blue</pl-answer>
+      <pl-answer>Pink</pl-answer>
+      <pl-answer>Purple</pl-answer>
+      <pl-answer>Orange</pl-answer>
+      <pl-answer>Yellow</pl-answer>
+      <pl-answer>Brown</pl-answer>
+      <pl-answer>Red</pl-answer>
+    </pl-multiple-choice>
+  </div>
 </div>
 
 <div class="card my-2">
-    <div class="card-header">
-        Part 2
-    </div>
-
-    <div class="card-body">
-
-        <pl-question-panel>
-            <p>Here, the <code>pl-multiple-choice</code> element is restricted to displaying <b>4</b> choices with one of the choices being the correct answer.
-            In addition, the choices are placed "inline" instead of being displayed
-            within a list. </p>
-
-            <p> What is the color of the sky? </p>
-        </pl-question-panel>
-
-        <pl-multiple-choice answers-name="sky-color2" number-answers="4" inline="true">
-          <pl-answer correct="true">Blue</pl-answer>
-          <pl-answer>Pink</pl-answer>
-          <pl-answer>Purple</pl-answer>
-          <pl-answer>Orange</pl-answer>
-          <pl-answer>Yellow</pl-answer>
-          <pl-answer>Brown</pl-answer>
-          <pl-answer>Red</pl-answer>
-        </pl-multiple-choice>
-
-    </div>
+  <div class="card-header">Part 2</div>
+
+  <div class="card-body">
+    <pl-question-panel>
+      <p>
+        Here, the <code>pl-multiple-choice</code> element is restricted to displaying
+        <b>4</b> choices with one of the choices being the correct answer. In addition, the choices
+        are placed "inline" instead of being displayed within a list. We also set
+        <code>hide-score-badge="True"</code> to hide the partial score percentage and badge.
+      </p>
+
+      <p>What is the color of the sky?</p>
+    </pl-question-panel>
+
+    <pl-multiple-choice
+      answers-name="sky-color2"
+      number-answers="4"
+      inline="true"
```

Comment:
```suggestion
      display="inline"
```
Unless you want this to showcase the old behaviour, in that case I believe you should keep this option in the docs.

Reply:
Changed here: https://github.com/PrairieLearn/PrairieLearn/pull/8903/commits/43bdad261b930462316f8fd5e00b162f2b040978